client/marklogic-types.d.ts

```diff
@@ -8,6 +8,8 @@ declare module 'marklogic' {
     }
 
     export type contentType = 'application/json' | 'application/xml' | 'text/html' | 'text/csv'
+    export type ResponseFormat = 'json' | 'xml' | 'csv'
```

Comment:
Since you have `RowsOptions` below - which is good, since it's specific to the rows endpoint - I think these should be `RowsResponseFormat` and `RowsQueryType` since both are specific to the rows endpoint as well.